wasmbinding/encoder.go

```diff
@@ -24,6 +29,14 @@ func CustomEncoder(sender sdk.AccAddress, msg json.RawMessage) ([]sdk.Msg, error
 		return dexwasm.EncodeDexPlaceOrders(parsedMessage.PlaceOrders)
 	case parsedMessage.CancelOrders != nil:
 		return dexwasm.EncodeDexCancelOrders(parsedMessage.CancelOrders)
+	case parsedMessage.CreateDenom != nil:
```

Comment:
LGTM

do we need to add in logic for tokenfactory queries as well?

Reply:
Discussed this on discord, it seems like we may want to the query of all the denoms created by an admin? Can include that in a separate PR if not high priority